src/server/config/config.js

```diff
@@ -163,8 +186,23 @@ module.exports = class Config {
 
   getSchema() {
     if (!this[schema]) {
-      let objKeys = zipObject([...this[schemaKeys]]);
-      this[schema] = Joi.object().keys(objKeys).default();
+      this[schema] = (function convertToSchema(children) {
+        let objKeys = Object.keys(children);
+        let schema = Joi.object().keys(objKeys).default();
```

Comment:
passing `objKeys` here is unnecessary. Joi is itterating the own properties of `objKeys` and adding them as keys, but since it's an array it has no own properties and does nothing.